src/test/rustdoc-gui/scrape-examples-fonts.goml

```diff
@@ -0,0 +1,18 @@
+goto: "file://" + |DOC_PATH| + "/scrape_examples/fn.test_many.html"
+
+define-function: (
```

Comment:
Is there still a point of having a function if it only contains one instruction?